examples/demo_integrate_limits.py

```diff
@@ -46,33 +46,34 @@
             ("ph2" , nddata(r_[0:4] / 4.0, "ph2")),
             ("t2" , nddata(r_[0:0.2:256j]-echo_time, "t2"))]),
             {"ph1": 0, "ph2": 1},
-            scale=10.)
-    data.setaxis('t2', lambda x: x-echo_time).register_axis({"t2":0}) # this
+            scale=20.)
+    # {{{ just have the data phase (not testing phasing here)
+    data.setaxis('t2', lambda x: x-echo_time).register_axis({"t2":0})
+    data = data['t2',0:-3] # dropping the last couple points avoids aliasing
+    #                        effects from the axis registration
+    #                        (otherwise, we get "droop" of the baseline)
```

Comment:
something I noticed while being picky -- might be best to roll this into the register_axis function -- I'm not sure.